hack/verify-flags/exceptions.txt

```diff
@@ -17,6 +17,8 @@ cluster/gce/coreos/helper.sh:# cloud_config yaml file should be passed
 cluster/gce/gci/configure-helper.sh:      reconcile_cidr="false"
 cluster/gce/gci/configure-helper.sh:  local api_servers="--master=https://${KUBERNETES_MASTER_NAME}"
 cluster/gce/gci/configure-helper.sh:  local reconcile_cidr="true"
+cluster/gce/gci/configure-helper.sh:  params+=" --enable_garbage_collector=${ENABLE_GARBAGE_COLLECTOR}"
```

Comment:
if you change the name to use dashes instead of underscore it won't be necessary.